src/main/java/com/checkout/DefaultCheckoutConfiguration.java

```diff
@@ -13,32 +13,29 @@
     private final URI baseUri;
     private final HttpClientBuilder httpClientBuilder;
     private final Executor executor;
-    private final FilesApiConfiguration filesApiConfiguration;
+    private Environment environment;
```

Comment:
I'm concerned with this change. So, this is not a deprecation, it seems to be a deprecation + actual removal of code. The idea is to add the new implementation while deprecating the old one (but make use of it in case it is set). Is this possible from your point of view?